cmd/create/machinepool/machinepool.go

```diff
@@ -22,6 +22,12 @@ import (
 func addMachinePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster, r *rosa.Runtime) {
 	var err error
 
+	isVersionSet := cmd.Flags().Changed("version")
+	if isVersionSet {
+		r.Reporter.Errorf("Setting `version` flag is not supported on classic rosa clusters")
```

Comment:
Should this not be available? Is it not possible? Just thinking right now

Reply:
Thanks for the comment, I haven't seen this parameter in the classic machinepool, as I introduced it only for the hosted ones (nodepools) I think that if a user tries to use it in that context it should get some error, but I can remove it to not introduce changes in the UX for classic flow as well.